auv_sim/auv_sim_description/auv_common_sim_descriptions/urdf/sensors/sonar.urdf.xacro

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 <robot name="sonar" xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:macro name="sonar_sim" params="name">
+  <xacro:macro name="sonar_sim" params="name topic">
     <gazebo reference="${name}_link">
       <sensor type="ray" name="${name}">
         <pose>0 0 0 0 0 0</pose>
         <visualize>false</visualize>
-        <update_rate>10</update_rate>
+        <update_rate>20</update_rate>
```

Comment:
This comment is totally unnecessary but in case if you wanted to further optimize your design, you can obtain this update rate as parameter.